components/developer/jenkins-core-lts/Makefile

```diff
@@ -41,7 +41,7 @@ COMPONENT_ARCHIVE=			$(COMPONENT_NAME)-$(JENKINS_RELEASE)-$(COMPONENT_VERSION).w
 COMPONENT_FMRI=				developer/jenkins-core-$(JENKINS_RELEASE)
 COMPONENT_CLASSIFICATION=	Development/Java
 COMPONENT_SRC=				$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_PROJECT_URL=		https://jenkins.io/changelog/
+COMPONENT_PROJECT_URL=		https://jenkins.io/changelog-stable/
 COMPONENT_ARCHIVE_URL=		https://get.jenkins.io/war-stable/$(COMPONENT_VERSION)/jenkins.war
 COMPONENT_LICENSE=			MIT
 COMPONENT_LICENSE_FILE= 	jenkins.license
```

Comment:
Making changes to a Makefile will trigger our jenkins to build a new artifact. This means that you either need to change COMPONENT_VERSION or COMPONENT_REVISION. Thus, please add COMPONENT_REVISION= 1 to the Makefile.

Reply:
Added.